tw-tasks-core/src/main/resources/db/changelog/db.tw-tasks-mysql.xml

```diff
@@ -1,41 +1,47 @@
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
 
   <changeSet id="14.09.2020.1" author="Kristo Kuusküll" runOnChange="false">
     <sql>
-      CREATE TABLE tw_task (
-      id BINARY(16) PRIMARY KEY NOT NULL,
-      status ENUM('NEW', 'WAITING', 'SUBMITTED', 'PROCESSING', 'DONE', 'ERROR', 'FAILED'),
-      -- Microsecond precision (6) is strongly recommended here to reduce the chance of gap locks deadlocking on tw_task_idx1
-      next_event_time DATETIME(6) NOT NULL,
-      state_time DATETIME(6) NOT NULL,
-      version BIGINT NOT NULL,
-      priority INT NOT NULL DEFAULT 5,
-      processing_start_time DATETIME(6) NULL,
-      processing_tries_count BIGINT NOT NULL,
-      time_created DATETIME(6) NOT NULL,
-      time_updated DATETIME(6) NOT NULL,
-      type VARCHAR(250) CHARACTER SET latin1 NOT NULL,
-      sub_type VARCHAR(250) CHARACTER SET latin1 NULL,
-      processing_client_id VARCHAR(250) CHARACTER SET latin1 NULL,
-      data LONGTEXT NOT NULL);
+      CREATE TABLE tw_task
+      (
+        id                     BINARY(16) PRIMARY KEY NOT NULL,
+        status                 ENUM('NEW', 'WAITING', 'SUBMITTED', 'PROCESSING', 'DONE', 'ERROR', 'FAILED'),
+        -- Microsecond precision (6) is strongly recommended here to reduce the chance of gap locks deadlocking on tw_task_idx1
+        next_event_time        DATETIME(6) NOT NULL,
+        state_time             DATETIME(6) NOT NULL,
+        version                BIGINT                            NOT NULL,
+        priority               INT                               NOT NULL DEFAULT 5,
+        processing_start_time  DATETIME(6) NULL,
+        processing_tries_count BIGINT                            NOT NULL,
+        time_created           DATETIME(6) NOT NULL,
+        time_updated           DATETIME(6) NOT NULL,
+        type                   VARCHAR(250) CHARACTER SET latin1 NOT NULL,
+        sub_type               VARCHAR(250) CHARACTER SET latin1 NULL,
+        processing_client_id   VARCHAR(250) CHARACTER SET latin1 NULL,
+        data                   LONGTEXT                          NOT NULL
+      );
 
       CREATE INDEX tw_task_idx1 ON tw_task (status, next_event_time);
 
-      CREATE TABLE tw_task_data (
-      task_id BINARY(16) PRIMARY KEY NOT NULL,
-      data_format INT NOT NULL,
-      data LONGBLOB NOT NULL
+      CREATE TABLE tw_task_data
+      (
+        task_id             BINARY(16) PRIMARY KEY NOT NULL,
+        data_format         INT      NOT NULL,
+        data                LONGBLOB NOT NULL,
+        task_context_format SMALLINT,
```

Comment:
Ideally the fixed length fields should be at the beginning of the table.
But it does not matter too much.

Besides we can probably only add columns to the end.